app/webhook.go

```diff
@@ -439,6 +464,8 @@ func (a *App) CreateOutgoingWebhook(hook *model.OutgoingWebhook) (*model.Outgoin
 		}
 	}
 
+	hook.SecretToken = model.NewRandomString(26)
```

Comment:
How would a user add a secret token to an existing webhook?

Reply:
For the existing webhooks, it would be part of updating the webhooks. There also will be a regeneration of the secret token as part webhook update.

Reply:
I think I was expecting some changes to the `UpdateOutgoingWebhook` method to support this -- or does this updating happen some other way?

Reply:
Sorry, I thought you'd want to add the feature in a separate pr and haven't added it yet. I will do so in the up-coming commit.

Reply:
Actually, it's wrong to generate `SecretToken` for IncomingWebhook because it should be generated by sender and input from webhook admin :) I will change it in the subsequent commit.